shadowfinder/cli.py

```diff
@@ -43,5 +56,34 @@ def find(
             raise ValueError(f"Invalid argument type or format: {e}")
         _validate_args(object_height, shadow_length, date_time)
 
-        shadow_finder = ShadowFinder(object_height, shadow_length, date_time)
+        shadow_finder = ShadowFinder(
+            object_height, shadow_length, date_time, time_format
+        )
+        shadow_finder.quick_find()
+
+    @staticmethod
+    def find_sun(
+        sun_altitude_angle: float,
+        date: str,
+        time: str,
+        time_format: str = "utc",
+    ) -> None:
+        """
+        Find the shadow length of an object given its height and the date and time.
```

Comment:
Switch to: "Locate a shadow based on the solar altitude angle and the date and time"